### ISSUE TITLE
User management

### ISSUE THREAD

**Issue by nomoregrapes** [peterwebster/henson#105]
The admin are doesn't yet show a list of users. Nor can you make other users become admins (or remove them)

**Comment by peterwebster** [comment]
Before @nomoregrapes leaves us, Katie and I need to have logged in using Twitter, and me using Wordpress.

**Comment by nomoregrapes** [comment]
I've now added the most simple page to list users, and links to make a user an "admin" or remove them.
Only admins can access this page, admins have all the special access to editing/ingesting/etc.
![admin-users](https://user-images.githubusercontent.com/1181194/41982242-d39855bc-7a22-11e8-89e1-a8170feedb8f.png)


Any type of user (logged in via Github or Twitter) can be made into an admin.
Logging in via Github is actually preferred, because it's needed if you want to export TSV and/or XML data to Github. Changes are then linked to your account ([example](https://github.com/nomoregrapes/test-files/commits/master)).

Closing this ticket as the work is done, and user management will be noted in documentation later.
